Description: Disable function mpg123_noise that is not yet part of
 libmpg123's official API and should therefore not get exported.
Author: Daniel Kobras <kobras@debian.org>
---
 src/libmpg123/dither.c |    2 ++
 src/libmpg123/dither.h |    2 ++
 2 files changed, 4 insertions(+)

--- mpg123.orig/src/libmpg123/dither.h
+++ mpg123/src/libmpg123/dither.h
@@ -17,7 +17,9 @@ enum mpg123_noise_type
 	,mpg123_highpass_tpdf_noise
 };
 
+#if 0
 void mpg123_noise(float* table, size_t count, enum mpg123_noise_type noisetype);
+#endif
 void dither_table_init(float *dithertable);
 
 #endif
--- mpg123.orig/src/libmpg123/dither.c
+++ mpg123/src/libmpg123/dither.c
@@ -100,6 +100,7 @@ static void highpass_tpdf_noise(float *t
 	}
 }
 
+#if 0
 void mpg123_noise(float* table, size_t count, enum mpg123_noise_type noisetype)
 {
 	switch(noisetype)
@@ -111,6 +112,7 @@ void mpg123_noise(float* table, size_t c
 		break;
 	}
 }
+#endif
 
 /* Generate white noise and shape it with a high pass filter. */
 void dither_table_init(float *dithertable)
